viewing_party/party.py

```diff
@@ -1,23 +1,123 @@
 # ------------- WAVE 1 --------------------
 
+from hashlib import new
+from re import M
+from collections import Counter
+from viewing_party.helper_functions import *
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1, USER_DATA_3
+
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        new_movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return new_movie
+    else:
+        return None

```

Comment:
Great job with this method! Couple of small callouts here.

You could convey intent a little more explicitly by pulling "return None" witha  guard clause with (on line 11 so it's the first line of code that executes in this method):
``` python
if not title or not genre or not rating:
      return None

# then proceed to create your movie dict
``` 
This way, when another developer comes along, they can immediately recognize "Oh  if these values aren't valid, then we'll return None. The way your code is currently structured, I can tell at some point this method will return None, but it's not as immediately clear.

Also a note about guard clauses - The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible. Read more [here](https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3)

You can also create your dictionary with literal syntax and return it without first creating the variable new_movie:
```
if not title or not genre or not rating:
      return None

return {
        "title": title,
        "genre": genre,
        "rating": rating        
    }
```

So your function would immediately have a guard clause that returns none and then the next line of code would return a dictionary with your movie data.